src/Components/common/contentEntitlement/index.js

```diff
@@ -0,0 +1,51 @@
+import i18n from "../../../i18n";
+import { usePelcro } from "../../../hooks/usePelcro";
+import { notify } from "../../../SubComponents/Notification";
+
+export const init = () => {
+  const { whenSiteReady } = usePelcro.getStore();
+
+  whenSiteReady(() => {
```

Comment:
this should be `whenUserReady` instead, but since we're having [issues with it](https://app.asana.com/0/1199393268587978/1200615874783464), we're using `whenSiteReady` instead

Reply:
Thanks for documenting that in a task!